lib/c-api/tests/wasmer-c-api-test-runner/src/lib.rs

```diff
@@ -316,46 +316,46 @@ fn test_ok() {
     }
 }
 
-#[cfg(test)]
```

Comment:
Since I was fixing errors across the code, I went in and fixed all of the failing clippy lints (for the default config at least). This was unused, so I commented it out, seeing as usages of it were also commented out.